package.json

```diff
@@ -1,40 +1,41 @@
 {
   "devDependencies": {
-    "@babel/core": "^7.0.0",
-    "@babel/node": "^7.0.0",
-    "@babel/plugin-proposal-class-properties": "^7.0.0",
-    "@babel/plugin-proposal-optional-chaining": "^7.0.0",
-    "@babel/plugin-transform-runtime": "^7.0.0",
-    "@babel/preset-env": "^7.0.0",
-    "@babel/preset-flow": "^7.0.0",
-    "@babel/preset-react": "^7.0.0",
-    "@babel/runtime": "^7.0.0",
+    "@babel/core": "7.0.0",
```

Comment:
Is this solving something you were running into, i.e. an npm vs. yarn issue?

Reply:
No not really.. but pinned versions have less surprises..

Reply:
fair enough..

Reply:
@KyleAMathews I reverted them